.github/workflows/test_and_coverage_server.yml

```diff
@@ -2,7 +2,6 @@ name: Server Integration Testing
 
 on: 
   pull_request:
-    branches: [ develop, master ]
```

Comment:
We also need to push new docker images no? 
- So if we merge something in develop, we would tag it with the commit sha but remove the "latest" tag,
- and then if we merge into master we would rebuild the same image but tag it with the "latest" tag

Does that make sense?

Reply:
Yes, that seems good to me